task_14/js/task3.js

```diff
@@ -0,0 +1,8 @@
+let day = prompt('введите количество дней');
+let second;
+if (day > 27 && day < 32){
```

Comment:
Hmmm, I also want to know how many seconds in 2, 3, 10 and etc. days)

Reply:
done